airflow/www/views.py

```diff
@@ -704,6 +704,12 @@ def render_template(self, *args, **kwargs):
         if TriggererJobRunner.is_needed():
             kwargs["triggerer_job"] = lazy_object_proxy.Proxy(TriggererJobRunner.most_recent_job)
 
+        executor, _ = ExecutorLoader.import_default_executor_cls()
+        try:
+            executor().check_health()
```

Comment:
I do not like this.  I feel like there must be a way to get the current health without instantiating a new executor class.